profiler/src/main/java/com/splunk/opentelemetry/profiler/allocation/sampler/AllocationEventSampler.java

```diff
@@ -17,17 +17,14 @@
 package com.splunk.opentelemetry.profiler.allocation.sampler;
 
 import java.util.function.BiConsumer;
-import jdk.jfr.consumer.RecordedEvent;
 
 /** A sampler that decides whether given allocation event should be sampled or discarded. */
```

Comment:
With the api change here this comment isn't really accurate any longer, and the name of the class is probably inaccurate too. I'm not sure what to call it thought, maybe just `SamplerStrategy` or something?

Reply:
I think I'll add the event argument back